### ISSUE TITLE
Should we keep the PR link in the title?

### ISSUE THREAD

**Issue by annevk** [whatwg/meta#128]
If we don't do this, the link is only available through proprietary GitHub metadata that GitHub does expose, but might not survive long term?

Also, squash + merge encourages including this and WPT folks encourage including this.

**Comment by domenic** [comment]
My take is that the PR link is only valuable as long as GitHub survives. And as long as GitHub survives, the metadata should survive. So I would rather not have it also in the commit message, especially since it's noisy when reading commit logs.

**Comment by annevk** [comment]
Yeah, I guess I'd be more supportive if this were put in the commit body as scanning commit titles with this in them is indeed less than ideal. Though again, only works if everyone enforces this...

**Comment by jgraham** [comment]
If wpt recommends this I think it shouldn't. It's strictly unnecessary for wpt since each commit that's ever the `HEAD` of master is tagged as `merge_pr_<number>` so we have the metadata available without needing to query GitHub or mandate specific author behaviour.
